mediawiki/mediawikipage.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, mediawiki, title=None, pageid=None, redirect=True,
         self._images = False
         self._references = False
         self._categories = False
+        self._langlinks = False
```

Comment:
I think that None would be better for representing non-fetched data, but I didn't want to break your convention.
Shall we consider changing this in the future?

Reply:
It is something I have considered in the past, but not something I want to deal with in this PR!

Reply:
Of course. I've also thought that this should be handled in different PR.